playwright/README.md

```
@@ -35,6 +35,12 @@ Run the below command to see playwright autogenerated report after passing test
     yarn playwright show-report
 ```
 
+#### Simulate slow network (CI-like conditions)
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)